hs_core/templates/pages/genericresource.html

```diff
@@ -99,10 +99,10 @@
 	"license": {
 	    "@type": "CreativeWork",
 	    {% if cm.metadata.rights.statement and cm.metadata.rights.url %}
-        "text": "{{ cm.metadata.rights.statement }}",
+        "text": "{{ cm.metadata.rights.statement | escapejs }}",
```

Comment:
Are there any other fields that should be escaped?

Reply:
https://github.com/hydroshare/hydroshare/issues/4184#issuecomment-811473710

Reply:
Good question. Abstracts/description field was already escaped in the initial implementation. I checked all other fields and verified this license statement field is the only other field that could have this issue.